### PR TITLE
Additional QAS Updates

### PR DESCRIPTION
**Pull request by edwardstamps**

### CODE REVIEW

File: web/admin/src/editCell.js [hunk 1]
```diff
@@ -9,16 +9,23 @@ export class CellEdit extends Component {
         this.state = {
             action : "state",
             value : this.props.task.sessionName,
-            focusBool: false
+            focusBool: false,
+            modalMessage: "",
+            height: 0,
+            padding: 0,
```
Comment (AdamLiechty) [inline web/admin/src/editCell.js]: padding and margin don't appear to be used

File: web/admin/src/editCell.js [hunk 2]
```diff
@@ -84,21 +88,24 @@ export class CellEdit extends Component {
             for (var item of this.props.sessions){
                 if (item.sessionName === named){
                   status = false
```
Comment (AdamLiechty) [inline web/admin/src/editCell.js]: This and the next 3 lines can be replaced with `status = (item.sessionName === this.props.task.sessionName)`

File: web/admin/src/header.js [hunk 3]
```diff
@@ -20,17 +20,31 @@ export class CustomHeader extends Component {
       var answeredQuestions = newQuestions.filter(item => item.answered === true)
           
       if (showBlock === false && showAnswer === false){
+        if (approveQuestions.length === 0){
```
Comment (AdamLiechty) [inline web/admin/src/header.js]: It's a bit hard to follow this nested `if` with the inconsistent tabbing

File: web/admin/src/headeroff.js [hunk 4]
```diff
@@ -20,26 +20,43 @@ export class CustomHeaderOff extends Component {
...
+              <span className="buttonSpan2">
+                <button className="listButton">Approved ({origQuestions.length})</button>
+                <button className="listButton2" onClick={this.props.handleClick}>Blocked ({blockedQuestions.length})</button>
+                <button className="listButton2"  onClick={this.props.handleAnswer}>Answered ({answeredQuestions.length})</button>
+                <span className="spacer"/>
+              </span>
+            )
+
+          }
+          else {
```
Comment (AdamLiechty) [inline web/admin/src/headeroff.js]: tabbing issues here as well make this confusing

File: web/admin/src/editCell.js (hunk 2 above)
Comment (edwardstamps) [reply web/admin/src/editCell.js]: change from ```if (item.sessionName === named){``` ?

Comment (AdamLiechty) [reply web/admin/src/editCell.js]: Yes,
```javascript
status = false
if (item.sessionName === this.props.task.sessionName){
  status = true
}
```

can be replaced with `status = (item.sessionName === this.props.task.sessionName)` since they will do the same thing
